doc/man/man3/crypto_kex_x_init_client.3monocypher

```diff
@@ -90,9 +89,10 @@ See
 .Xr intro 3monocypher
 for advice about generating random bytes (use the operating system's
 random number generator).
+This argument is not constant because it is wiped after use.
 .It Fa local_sk
 A 32-byte secret random number.
-This should be different for every connection you make.
+This is a long-term secret used to identify the respective party.
```

Comment:
Nitpick: identification is done with the *public* half. Not sure how to phrase it, though, this one is probably good enough. Let's leave it at that for now and ask @mikejsavage instead.